friendships/services.py

```diff
@@ -72,5 +79,20 @@ def get_followers(cls, user_id):
         return followers
 
     @classmethod
-    def get_has_followed(cls, from_user, to_user):
-        return Friendship.objects.filter(from_user=from_user, to_user=to_user).exists()
+    def get_following_user_id_set(cls, from_user_id):
+        key = FOLLOWINGS_PATTERN.format(user_id=from_user_id)
+        following_user_id_set = cache.get(key)
+        if following_user_id_set is not None:
+            return following_user_id_set
```

Comment:
should call get_following_user_id_set() twice to test cache.
Get followings from DB and set to cache first time, and get followings from cache second time

Reply:
Resolved as suggestion